exporter/stackdriver/stats.go

```diff
@@ -348,6 +350,10 @@ func newTypedValue(vd *view.View, r *view.Row) *monitoringpb.TypedValue {
 				BucketCounts: v.CountPerBucket,
 			},
 		}}
+	case *view.LastValueData:
+		return &monitoringpb.TypedValue{Value: &monitoringpb.TypedValue_DoubleValue{
+			DoubleValue: float64(v.Value),
```

Comment:
Is this case necessary? Isn't this already a float64?

Reply:
done